src/components/CaptureDetailDialog.js

```diff
@@ -116,49 +139,65 @@ function CaptureDetailDialog(props) {
-            value: capture.planterIdentifier,
-            copy: true,
-          },
-          {
-            label: 'Device Identifier',
-            value: capture.deviceIdentifier,
-            copy: true,
-          },
-          { label: 'Approved', value: capture.approved ? 'true' : 'false' },
-          { label: 'Active', value: capture.active ? 'true' : 'false' },
-          { label: 'Status', value: capture.status },
-          { label: 'Species', value: species && species.name },
-          { label: 'Created', value: dateCreated.toLocaleString() },
-          { label: 'Note', value: renderCapture.note },
-        ].map((item) => (
-          <Fragment key={item.label}>
-            <Grid item>
-              <Typography variant="subtitle1">{item.label}</Typography>
-              <Typography variant="body1">
-                {item.link ? (
-                  <LinkToWebmap value={item.value} type="user" />
-                ) : (
-                  item.value || '---'
-                )}
-                {item.value && item.copy && (
-                  <CopyButton label={item.label} value={item.value} />
-                )}
-              </Typography>
-            </Grid>
-            <Divider />
-          </Fragment>
-        ))}
         <Grid item>
-          <Typography variant="subtitle1">Tags</Typography>
+          <Typography className={classes.subtitle}>Capture Data</Typography>
+          {[
+            {
+              label: 'Planter ID',
+              value: capture.planterId,
+              copy: true,
+              link: true,
+            },
+            {
+              label: 'Planter Identifier',
+              value: capture.planterIdentifier,
+              copy: true,
+            },
+            {
+              label: 'Device Identifier',
+              value: capture.deviceIdentifier,
+              copy: true,
+            },
+            { label: 'Species', value: species && species.name },
+            { label: 'Created', value: dateCreated.toLocaleString() },
+            { label: 'Note', value: renderCapture.note },
+          ].map((item) => (
+            <Fragment key={item.label}>
+              <Grid item>
+                <Typography variant="subtitle1">{item.label}</Typography>
+                <Typography variant="body1">
+                  {item.link ? (
+                    <LinkToWebmap value={item.value} type="user" />
+                  ) : (
+                    item.value || '---'
+                  )}
+                  {item.value && item.copy && (
+                    <CopyButton label={item.label} value={item.value} />
+                  )}
+                </Typography>
+              </Grid>
+            </Fragment>
+          ))}
+        </Grid>
+        <Divider />
+        <Grid item>
+          <Typography className={classes.subtitle}>
+            Verification State
+          </Typography>
+          {!capture.approved && capture.active ? (
```

Comment:
I'd suggest creating a small helper function to translate `(active, approved)` into a verification status to prevent duplication of this logic and make the relationship more obvious.